### DISCUSSION TITLE
The FASTA file for Lib-Free search: with or without decoys?

### DISCUSSION THREAD

**Discussion by dorvashmr** [vdemichev/DiaNN#569] [General]
Hi all,

I have a really newbie question that could not find the answer to in the documentation, nor by searching and surfing through the community discussions.
So, for the library-free search (or in general, for having a FASTA file in your DIA-NN search), should we include the decoys in the database in advance, or does DIA-NN do it for us? 

Cheers,
Reza

**Comment by vdemichev** [comment]
Hi Reza,

No, DIA-NN generates decoys automatically and then should never be included.

Best,
Vadim

**Comment by dorvashmr** [comment]
Thank you Vadim. I guess I realized now why some of my searches are failing.

Cheers,
Reza
